src/content/tutorial/react-step-5/react-step-5.mdx

```diff
@@ -3,68 +3,216 @@ title: 5. Deploying to IBM Cloud
+
+##### src/index.scss
+
+```scss
+// Feature flags
+$css--font-face: true;
+$css--plex: true;
+
+// Global styles
+@import 'carbon-components/scss/globals/scss/css--font-face';
+@import 'carbon-components/scss/globals/grid/grid';
+
+// Carbon components
+@import 'carbon-components/scss/components/breadcrumb/breadcrumb';
+@import 'carbon-components/scss/components/button/button';
+@import 'carbon-components/scss/components/data-table/data-table';
+@import 'carbon-components/scss/components/link/link';
+@import 'carbon-components/scss/components/pagination/pagination';
+@import 'carbon-components/scss/components/tabs/tabs';
+@import 'carbon-components/scss/components/ui-shell/ui-shell';
+```
+
+In comparing to the included `styles.scss`, you may be asking what happened to importing `_vars.scss`, `_colors.scss`, `_theme.scss`, etc.? Many of those global Sass partials get imported through the components. For example, open `node_modules/carbon-components/scss/components/button/_button.scss` to see its dependencies. No harm in importing them as `styles.scss` does, but for simplicity here, we'll let the components pull them in.
+
+You can read more about optimizing Carbon's Sass in the [Carbon Design System publication](https://medium.com/carbondesign/minimal-css-with-carbon-b0c089ccfa71) on Medium.
+
+## Build for production
+
+Before we deploy our app, we need create an optimized production build with this command. You may need to `CTRL-C` to stop the development environment first.
 
 ```bash
-yarn build
+$ yarn build
 ```
 
-### Create manifest file
+Looking at `package.json`, you'll find `yarn build` to run `react-scripts build`. This builds the app for production to the `build` folder. It bundles React in production mode and optimizes the build for the best performance. It even goes so far to minify files and include hashes in filenames for caching.
+
+As a lot of this may seem like magic since the build configuration came from Create React App, go ahead and check out their [production build guidelines](https://facebook.github.io/create-react-app/docs/production-build) for a full description of what's happening.
+
+## Create manifest file
+
+Now that we have a production build, let's get it on the cloud. We're going to use [staticfile-buildpack](https://github.com/cloudfoundry/staticfile-buildpack.git) to deploy our webapp. Since this is a Cloud Foundry buildpack, we'll be using the `cf` command line interface (CLI). If running `cf --help` doesn't work for you, chances are you need to [install the CLI](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html).
+
+_Note: If unfamiliar with buildpacks, the [staticfile buildpack docs](https://docs.cloudfoundry.org/buildpacks/staticfile/index.html) has good definitions and configuration documentation._
+
+With the Cloud Foundry CLI installed, next we need to create a `manifest.yml` file in the root of the project. You'll need to add your `REACT_APP_GITHUB_PERSONAL_ACCESS_TOKEN` here.
```

Comment:
Should we add a link back to step 3 where we first have them create the token?

> With the Cloud Foundry CLI installed, next we need to create a `manifest.yml` file in the root of the project. You'll need to add your `REACT_APP_GITHUB_PERSONAL_ACCESS_TOKEN` that you created in [Step 3](https://www.carbondesignsystem.com/tutorial/react-step-3#create-access-token) here.

Reply:
Will do.